### PR TITLE
[Fix] PagerDuty is python2 but yml says python3

### PR DESCRIPTION
**Pull request by DeanArbel**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/19409

## Description
1) Reverted the integration subtype to `subtype: python2`.
2) Fixed unicode errors experienced by some customers.
3) Made integration python 3 compatible.

## Does it break backward compatibility?
   - Yes (and no) - Changed the subtype back to `python2`

## Must have
- [x] Tests
- [x] Documentation (with link to it)
- [ ] Code Review

## Technical writer review
Mention and link to the files that require a technical writer review.
- [ ] [CHANGELOG](https://github.com/demisto/content/edit/pagerduty-fixes/Integrations/PagerDuty/CHANGELOG.md?pr=%2Fdemisto%2Fcontent%2Fpull%2F4509)

### CODE REVIEW

**Comment by DeanArbel** [comment]
Customer confirmed this fixed their issues (including unicode issues)

**Comment by DeanArbel** [comment]
@ronykoz please note the build is passing, but fails on script validation because of the change to `subtype`

**Comment by DeanArbel** [comment]
@gal-berger fyi

**Comment by yaakovi** [comment]
@DeanArbel missing changelog

File: Integrations/PagerDuty/PagerDuty.py [hunk 1]
```diff
@@ -102,11 +103,11 @@ def http_request(method, url, params_dict=None, data=None):
                                )
         res.raise_for_status()
 
-        return res.json()
+        return unicode_to_str_recur(res.json())
 
     except Exception as e:
         LOG(e)
-        raise(e)
+        raise e
```
Comment (yaakovi) [inline Integrations/PagerDuty/PagerDuty.py]: remove the `e`
```suggestion
        raise
```

File: Integrations/PagerDuty/PagerDuty.py [hunk 2]
```diff
@@ -10,6 +10,7 @@
 requests.packages.urllib3.disable_warnings()
 
 ''' GLOBAL VARS '''
+IS_PY3 = sys.version_info[0] == 3
```
Comment (yaakovi) [inline Integrations/PagerDuty/PagerDuty.py]: there is already a global in CommonServerPython for this, use it.

File: Integrations/PagerDuty/PagerDuty.py (hunk 1 above)
Comment (DeanArbel) [reply Integrations/PagerDuty/PagerDuty.py]: > add changelog

@ronykoz asked to drop the RN I added.

**Comment by DeanArbel** [comment]
> add changelog

@ronykoz asked to drop the RN I added.

**Comment by yaakovi** [comment]
@DeanArbel trigger a build without BC checks. if it passes, we can merge it.

**Pull request merged**